src/apps/service/apps/combined.clj

```diff
@@ -365,4 +365,5 @@
     (.hasAppPermission (util/get-apps-client clients system-id) username system-id app-id required-level))
 
   (supportsJobSharing [_ job-step]
-    (.supportsJobSharing (util/apps-client-for-job-step clients job-step) job-step)))
+    (and (not= (:job_type job-step) jp/agave-job-type)
+         (.supportsJobSharing (util/apps-client-for-job-step clients job-step) job-step))))
```

Comment:
The changes in this namespace are only for allowing users to still list their old Agave jobs, even if they can't relaunch them anymore.

Maybe it would be better to re-add an `Agave` type that handles the logic in these methods, and throws an error for the rest? 🤷

Reply:
That seems pretty clean to me, but this is also understandable, so I'm open to leaving it as is.